### DISCUSSION TITLE
Angular unit test with JsStore : error

### DISCUSSION THREAD

**Discussion by Lempkin** [ujjwalguptaofficial/JsStore#272] [Q&A]
Hi,

I use jsstore in my angular app.
I try to set some unit tests that should insert/update/delete in DB (db must not be mocked).

My first unit test only instantiate service : 
`expect(service).toBeTruthy();`

In the service's constructor I run a select to get all items in db.
What I see is that this unit test seems to be ok, but then I get following error : 

> Chrome Headless 103.0.5060.66 (Windows 10) ERROR
>   Disconnected , because no message in 30000 ms.

I've tried to comment the select and only call the db connection and error still happen. Seems that as soon as I get a db connection, ng test will inevitably fails in the end with this error.

Any idea? 
Thx

**Comment by ujjwalguptaofficial** [comment]
not sure what's happening, something might be breaking check the console.
